packages/e2e-tests/specs/embedding.test.js

```diff
@@ -229,6 +231,21 @@ describe( 'Embedding content', () => {
 		await page.waitForSelector( 'figure.wp-block-embed-twitter' );
 	} );
 
+	it( 'embed blocks should work inside blocks with locking all', async () => {
+		await activatePlugin( 'gutenberg-test-innerblocks-locking-all-embed' );
+		await createNewPost();
+		await insertBlock( 'Test Inner Blocks Locking All Embed' );
+		const embedInputSelector = '.components-placeholder__input[aria-label="Embed URL"]';
+		await page.waitForSelector( embedInputSelector );
+		await page.click( embedInputSelector );
+		// This URL can't be embedded, but without the trailing slash, it can.
+		await page.keyboard.type( 'https://twitter.com/notnownikki' );
+		await page.keyboard.press( 'Enter' );
+		// The twitter block should appear correctly.
+		await page.waitForSelector( 'figure.wp-block-embed' );
+		await deactivatePlugin( 'gutenberg-test-innerblocks-locking-all-embed' );
```

Comment:
I think this test should be wrapped with its own `describe` and the plugin should be activated in `before` and deactivated in `after` methods to ensure they are properly handled when the test fails.